Homework/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Tyrone Biggums</title>
+		<link rel="stylesheet" href="css/styles.css">
+		<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"/>
+		<link rel="stylesheet" href="http://colorzilla.com/gradient-editor/#b4ddb4+0,83c783+17,52b152+33,008a00+67,005700">
+
+	</head>
+	<body>
+
+		<nav>
+			<a href="index.html">About Me |</a>
+			<a href="portfolio.html">Portfolio</a>
+			
+		</nav>
+
+		<h1 class="text-color">SHAZAM!!!</h1>
+		<hr>
```

Comment:
@qodesmith Ok, I think we had just gone over it in class that day, and I was just testing it out. I like the idea of adding a top border on the next element or using a <div> tag, that way I could have more control via CSS later. Thanks for the tip